contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -174,33 +175,6 @@ contract LoanTokenLogicStandard is AdvancedToken, GasTokenUser {
         );
     }
 
```

Comment:
Functions "borrow" and "marginTrade" are missing pausable. Was this intentional?